Georgeygigz/store-manager-Api-v2
# #161439600  Setup a PostgreSQL database

**What does this PR do?**
It sets up a new PostgreSQL database

**Description completed tasks**
1. Installing PostgreSQL database
2. Create a new connection with the help of psycopg2-binary 
3. Creating the required tables

**Test whether the connection work**
1. _Install a Postgres database_
2. _create a new database and export the database url to virtual env_
3. _`git clone` this [repo](https://github.com/Georgeygigz/store-manager-Api-v2)_
4. _Navigate to the project folder_
5. _Run the application and check in your data base whether new tables are created_

**PT Stories**
#161439600
